ops/charm.py

```diff
@@ -297,86 +245,70 @@ def restore(self, snapshot):
             self.unit = None
 
 
+class RelationCreatedEvent(RelationEvent):
+    """Represents the `relation-created` hook from Juju.
+
+    This is triggered whenever relations are created in the Juju model before relation-join
+    events. For units that are coming up, relation-created events fire after the install event
```

Comment:
The sentence still reads incorrectly. It's saying "this is triggered whenever A happens before B". This is simply not the case.

Reply:
https://github.com/canonical/operator/pull/218/commits/e375646f240307b42e5f95ee58a60cb2e65a5eb0 - here's an attempt at fixing it.